### ISSUE TITLE
haskell-language-server 0.5.1 for GHC 8.6.3 is not available on Linux.

### ISSUE THREAD

**Issue by snthibaud** [haskell/haskell-language-server#559]
If you encounter a bug or you have a support question, please try to fill out some of the information below.
However, if you think your issue does not need any of it, you may omit it.
Generally speaking, the information below is meant for helping debugging issues
but they are no prerequisite for opening issues.

### Subject of the issue
I encountered this error message installing the haskell-language-server plugin: haskell-language-server 0.5.1 for GHC 8.6.3 is not available on Linux.

### Your environment
haskell-language-server: コマンドが見つかりません
haskell-language-server-wrapper: コマンドが見つかりません

### Steps to reproduce
1. Install VS Code
2. Install haskell-language-server extension
3. Install 'Haskell GHCi Debug Adapter Phoityne' extension
4. Try to debug
(I am not sure if steps 3 and 4 were actually necessary)

### Expected behaviour
The message should not occur or should give more instructions on what to change. Should I use a different GHC version or a different haskell-language-server/Haskell extension version?

### Actual behaviour
A message indicates the problem, but there are no steps toward a solution.

**Comment by fendor** [comment]
Yeah, GHC 8.6.3 ist not supported by HLS, so you need to use a newer ghc version for your project.
